common/components/common/tags/TagSelectorCollapsible.jsx

```diff
@@ -94,6 +111,24 @@ class TagSelectorCollapsible extends React.Component<Props, State> {
     );
   }
 
+  _AllChecked(): void {
```

Comment:
Here's the big problem function right now. Aside from the name, which I should probably make better, what I want is something that will check or uncheck the Select All checkbox for a given category of filters based on whether or not all the filters are themselves checked.

so two main issues I've got right now - it's not updating correctly in the DOM so I need to fix that, and I also think it's going to run into problems with subcategories due to how those components are structured.